Framework/Algorithms/src/AddSampleLog.cpp

```diff
@@ -51,6 +60,13 @@ void AddSampleLog::exec() {
   std::string propValue = getProperty("LogText");
   std::string propUnit = getProperty("LogUnit");
   std::string propType = getPropertyValue("LogType");
+  std::string propNumberType = getPropertyValue("NumberType");
+
+  if (!propNumberType.empty() &&
```

Comment:
Might be better if that wen't into an information log instead. Maybe this is why you have the empty entry.